pkg/addons/velero/velero.go

```diff
@@ -163,6 +163,9 @@ func (o *Velero) Outro(ctx context.Context, cli client.Client) error {
 }
 
 // New creates a new Velero addon.
-func New(isEnabled bool) (*Velero, error) {
-	return &Velero{isEnabled: isEnabled}, nil
+func New(namespace string, isEnabled bool) (*Velero, error) {
+	return &Velero{
+		namespace: namespace,
+		isEnabled: isEnabled,
+	}, nil
 }
```

Comment:
is there a reason to allow configuring the velero namespace?

Reply:
I would rather have the entire set of changes in this file reverted honestly

Reply:
Not particularly, but I wanted to follow the pattern we have for the admin console addon.

Reply:
I think allowing us to configure the admin console namespace is also a mistake
so if one should change it's the admin console 😄